oauth2client/contrib/gce.py

```diff
@@ -48,28 +45,67 @@
 """
 
 
-def _get_service_account_email(http_request=None):
-    """Get the GCE service account email from the current environment.
-
+def _get_metadata(http_request=None, path=None, recursive=True):
```

Comment:
I'd guess the original author haphazardly threw this together, so basing on their impl. may not be useful. Also there was only one thing retrieved (the access token) and now there are many.

Can you throw together a list of all metadata URLs retrieved and all the values used from each?